pages/news.html

```diff
@@ -113,7 +114,7 @@ <h1>NEWS</h1>
             <p><strong>02/08/2019</strong> - Uyuni 4.0.2 is now available! This release enables Ubuntu clients, monitoring, content lifecycle management and <a href="../pages/stable-version.html#releasenotes">much more</a>!</p>
             <p><strong>05/04/2019</strong> - openSUSE Summit Nashville - <a href="https://events.opensuse.org/conferences/oSSN19/program/proposals/2276">Managed Monitoring with Prometheus, Grafana and Uyuni</a></p>
             <p><strong>08/03/2019</strong> - SCALE17x Presentation - <a href="https://www.socallinuxexpo.org/scale/17x/presentations/uyuni-adding-flavor-open-source-infrastructure-management">Uyuni: Adding flavor to open source infrastructure managementx</a></p>
-            
+
```

Comment:
Why is this needed?

Reply:
I was wondering about that myself. My editor doesn't see a change there, so maybe it's the GH diff? :woman_shrugging: